tests/e2e/vsan_stretched_cluster_utils.go

```diff
@@ -246,3 +248,115 @@ func createPvcInParallel(client clientset.Interface, namespace string, diskSize
 		lock.Unlock()
 	}
 }
+
+// siteNetworkFailure chooses a site to create or remove network failure
+func siteNetworkFailure(primarySite bool, removeNetworkFailure bool) {
+	hosts := fds.secondarySiteHosts
+	if primarySite {
+		hosts = fds.primarySiteHosts
+	}
+	fds.hostsPartitioned = hosts
```

Comment:
`fds.hostsPartitioned` should be set to hosts inside the else block not here and it should be cleared when hosts are no longer partitioned